build/commands/lib/config.js

```diff
@@ -100,7 +100,20 @@ const parseExtraInputs = (inputs, accumulator, callback) => {
   }
 }
 
+const getBraveVersion = (useFullVersion) => {
```

Comment:
We also have a code with `VERSION` parsing in `build.js`. Maybe we can combine all the related code in something like `lib/version.js`?

Reply:
> We also have a code with `VERSION` parsing in `build.js`. Maybe we can combine all the related code in something like `lib/version.js`?

`getBraveVersion()` is closely related to `package.json` and `packageConfig()`, not sure if it's the right thing to separate these.

I'll update `build.js` message be a little clearer when you enable/disable `use_full_version` flag.